### PR TITLE
Enhanced style control to ratings 

### PR DESCRIPTION
**Pull request by timotew**
I notice the ratings text comes with default colors so I make it more editable.
The rating ext doesn't align with the ratings so I fix that and I fixed a typo in the doc too.

```
//Before 

```

```
// After
     <Rating
        showRating
        type="star"
        fractions={1}
        startingValue={3.6}
        ratingTextColor={'white'}
        imageSize={20}
        onFinishRating={()=>{}}
        style={{ paddingVertical: 10 }}
      />
```

```
// Before
      <Rating
        showRating
        readonly
        type="star"
        fractions={1}
        startingValue={3.6}
        ratingTextColor={'white'}
        imageSize={20}
        onFinishRating={()=>{}}
        style={{ paddingVertical: 10 }}
      />
```

```
// After

   <Rating
        showRating
        readonly
        showReadOnlyText={false}
        type="star"
        fractions={1}
        startingValue={3.6}
        ratingTextColor={'white'}
        imageSize={20}
        onFinishRating={()=>{}}
        style={{ paddingVertical: 10 }}
      />
```

### CODE REVIEW

**Comment by timotew** [comment]
@Monte9 Hi there! Are these changes allowed?

**Comment by iRoachie** [comment]
Hey @timotew, right now we're in the process of preparing for a PR that will enable compatibility with react-native-web. As a result many of the components have had small alterations to make them ready for web support. These changes may or may not affect the file(s) you want to change in your PR.

So that we don't have any conflicts with the merge of that PR, we'll be temporarily not accepting PRs until the react-native-web PR has been merged. It should be done this week.

Thanks for understanding

**Comment by timotew** [comment]
This is coming to the web :+1:  great.
@iRoachie  are contributions allowed? or it's done.

**Comment by iRoachie** [comment]
Sure, you can follow the conversation here https://github.com/react-native-training/react-native-elements/pull/867

**Comment by Monte9** [comment]
@timotew with changes like these, please include a screenshot & a code snippet of how to use the additional props that you added and what effect they have.

**Comment by timotew** [comment]
@Monte9  You mean I should add it to the doc or the PR comment?

**Comment by Monte9** [comment]
To this PR! It will help us visualise the changes you have made and will make it easier to review.

**Comment by timotew** [comment]
ping @Monte9 .

**Comment by Monte9** [comment]
@timotew sorry for the delay. Change you fix the merge conflict and then I can take a look at this again,

**Comment by xavier-villelegier** [comment]
@timotew What's the status of this PR ? Feel free to update it or close it, let us know

**Comment by timotew** [comment]
@xavier-villelegier , @Monte9  I apologies for the delay. I will resolve the conflict ASAP. Thanks.

**Comment by timotew** [comment]
@Monte9, @xavier-villelegier   please take a look again.

File: src/rating/Rating.js [hunk 1]
```diff
@@ -337,7 +348,9 @@ Rating.propTypes = {
   onFinishRating: PropTypes.func,
   showRating: PropTypes.bool,
   style: ViewPropTypes.style,
+  ratingTextColor: PropTypes.string,
```
Comment (xavier-villelegier) [inline src/rating/Rating.js]: Add this props to the docs too

File: docs/rating.md [hunk 2]
```diff
@@ -94,6 +94,7 @@ const { rating } = this.props;
 | ratingColor           | #f1c40f         | string (color) | Pass in a custom fill-color for the rating icon; use this along with `type='custom'` prop above (optional)                   |
 | ratingBackgroundColor | white           | string (color) | Pass in a custom background-fill-color for the rating icon; use this along with `type='custom'` prop above (optional)        |
 | ratingCount           | 5               | number         | The number of rating images to display (optional)                                                                            |
+| ratingText            | #f1c40f         | string (color) | Pass in a custom text color for the rating text (optional)                                                                   |
```
Comment (iRoachie) [inline docs/rating.md]: It's ratingTextColor

**Pull request merged**